sdcm/cluster.py

```diff
@@ -5006,6 +5006,20 @@ def install_scylla_monitoring_prereqs(node):  # pylint: disable=invalid-name
                 pip3 install -I -U psutil
                 systemctl start docker
             """)
+        elif node.is_ubuntu():
```

Comment:
so here we will have Ubuntu 20.04 and Ubuntu 22.04... why not to unify all the 3 supported Ubuntu versions?

Reply:
Also, as we discussed face to face, please add the if statement of ubuntu 18 into is_ubunto and add a TODO comment to remove it once the we stop supporting Ubuntu 18